examples/edge-functions/supabase/functions/slack-bot-mention/index.ts

```diff
@@ -0,0 +1,45 @@
+import { serve } from 'https://deno.land/std@0.197.0/http/server.ts';
+import { WebClient } from 'https://deno.land/x/slack_web_api@6.7.2/mod.js';
+
+
+const slack_bot_token = Deno.env.get("SLACK_TOKEN") ?? "";
+const bot_client = new WebClient(slack_bot_token);
+
+console.log(`Slack URL verification function up and running!`);
+serve(async (req) => {
+  try {
+    const req_body = await req.json();
+    console.log(JSON.stringify(req_body, null, 2));
+    const { token, challenge, type, event } = req_body;
```

Comment:
For example purposes, should we remove the `console.log` and consolodate these into a single line?